traceapp/vis.go

```diff
@@ -14,6 +14,8 @@ import (
 	_ "sourcegraph.com/sourcegraph/appdash/sqltrace"
 )
 
+// ErrTimelineItemValidation is returned by timelineItem.Valid when either
+// timelineItem.Label or timelime.ItemFullLabel are empty.
 var ErrTimelineItemValidation = errors.New("timeline item validation error")
```

Comment:
Agreed, if it doesn't need to be exported, it's better to make it not exported. It makes the public API more readable by having fewer entries.